File: week3-app-development/javascript/Ex06_Connect_to_Astra.js
const connection = require('./db_connection')
const Uuid       = require('cassandra-driver').types.Uuid;

console.log("========================================")
console.log("Start exercise")

connection.client
.execute('SELECT * FROM system.local')
.then(function(result) {
	result.rows.forEach(row => {
        console.log("Your are now connected to Astra '%s' at '%s'", row.cluster_name, row.data_center)
    })
    connection.client.shutdown()
    console.log("SUCCESS")
})
.catch(function(error){
    console.log(error.message)
    connection.client.shutdown()
});

console.log("========================================")

File: week3-app-development/python/Ex06_Connect_to_Astra.py
#!/usr/bin/env python3
from connection import session

print('========================================')
print('Start exercise')
try:
    output = session.execute("SELECT * FROM system.local")
    for row in output:
        print('You are now connected to cluster %s at %s' %
              (row.cluster_name, row.data_center))
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
print('========================================')
